context.go

```diff
@@ -23,6 +44,16 @@ func newContext(c *Conode, o *Overlay, servID ServiceID, manager *serviceManager
 	}
 }
 
+// contextDataPath indicates where the service-data will be stored. If it is
+// empty, a memory-map is used.
+var contextDataPath = ""
+
+func init() {
+	if flag.Lookup("test.v") == nil {
```

Comment:
It doesn't matter, because even without the verbose flag, "test.v" is defined, but it is `false`. First time I like _go_ making the difference between `nil` and `false` ;)